website/content/docs/get-started/developer-qs.mdx

```diff
@@ -229,19 +303,52 @@ VaultTemplate vaultTemplate = new VaultTemplate(
 );
 ```
 
+</Tab>
+<Tab heading="Bash" group="bash">
+
 ```shell-session
 export VAULT_TOKEN="dev-only-token"
 ```
 
-</CodeTabs>
+</Tab>
+<Tab heading="OpenAPI Go (Beta)" group="openAPI-go">
```

Comment:
Hi! Could we get the C# examples here as well? They should be up to date in the [vault-client-dotnet](https://github.com/hashicorp/vault-client-dotnet) readme